resim/examples/sim_runner/test_run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#!/bin/bash
+
+set -e
+
+SIM_RUNNER_DIR=resim/examples/sim_runner
+
+# Set up the experience in local folders in a volume which the dev container #
+# mounts at /root/.
+
+INPUTS=/root/inputs
+OUTPUTS=/root/outputs
+
+mkdir -p "${INPUTS}" "${OUTPUTS}"
+cp "${SIM_RUNNER_DIR}/experience.sim" "${INPUTS}"
+
+# The ReRun worker will run a command similar to this, ensuring that
```

Comment:
This comment isn't totally clear to me

Reply:
Okay. I've tried to re-word this slightly to be more clear.